lib/sidekiq-status/web.rb

```diff
@@ -48,7 +48,17 @@ def has_sort_by?(value)
         jids = namespace_jids.map{|id_namespace| id_namespace.split(':').last }
         @statuses = []
 
-        jids.each do |jid|
+        if params[:max_items] == 'all'
+          @max_items = 'all' if params[:max_items] == 'all'
+          max_items = jids.size if @max_items == 'all'
+        else
+          is_a_valid_max_item_param = %w(20 50 100).include?(params[:max_items])
+          @max_items = 20 unless is_a_valid_max_item_param
+          @max_items ||= params[:max_items].to_i
+          max_items = @max_items
+        end
+
+        jids[0..max_items-1].each do |jid|
```

Comment:
To avoid a (potentially) expensive memory copy, could you please rewrite like the following?
```ruby
(0..max_items-1).each do |idx|
  jid = jids[idx]
  ...
```